### ISSUE TITLE
Python3.10 - collections module dependency

### ISSUE THREAD

**Issue by sngx13** [excentis/examples-byteblower-test-framework.py#1]
Hi,

Looks like dependency issue with 3rd party module "highcharts" where it's using **Iterable** from _collections_, now has become _collections.abc_ is there a workaround of not relying on this?

```
root@brlb-dev-vm-01:/opt/tests# ./byteblower.py
Traceback (most recent call last):
  File "/opt/tests/./byteblower.py", line 7, in <module>
    from byteblower_test_framework.analysis import HttpAnalyser
  File "/usr/local/lib/python3.10/dist-packages/byteblower_test_framework/analysis.py", line 7, in <module>
    from ._analysis.bufferanalyser import BufferAnalyser
  File "/usr/local/lib/python3.10/dist-packages/byteblower_test_framework/_analysis/bufferanalyser.py", line 7, in <module>
    from .plotting.generic_chart import GenericChart
  File "/usr/local/lib/python3.10/dist-packages/byteblower_test_framework/_analysis/plotting/__init__.py", line 1, in <module>
    from .generic_chart import GenericChart
  File "/usr/local/lib/python3.10/dist-packages/byteblower_test_framework/_analysis/plotting/generic_chart.py", line 4, in <module>
    from highcharts import Highchart
  File "/usr/local/lib/python3.10/dist-packages/highcharts/__init__.py", line 5, in <module>
    from .highcharts.highcharts import Highchart
  File "/usr/local/lib/python3.10/dist-packages/highcharts/highcharts/__init__.py", line 1, in <module>
    from .highcharts import Highchart
  File "/usr/local/lib/python3.10/dist-packages/highcharts/highcharts/highcharts.py", line 15, in <module>
    from collections import Iterable
ImportError: cannot import name 'Iterable' from 'collections' (/usr/lib/python3.10/collections/__init__.py)
root@brlb-dev-vm-01:/opt/tests#
root@brlb-dev-vm-01:/opt/tests#
root@brlb-dev-vm-01:/opt/tests# python3 --version
Python 3.10.6
root@brlb-dev-vm-01:/opt/tests# python3
Python 3.10.6 (main, Nov 14 2022, 16:10:14) [GCC 11.3.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> from collections.abc import Iterable
>>>

```
Thank you

**Comment by tomghyselinck** [comment]
Hi Serghei,

Thank you for notifying about this issue!

It is indeed a known issue with one of our Python library dependencies ([python-highcharts](https://pypi.org/project/python-highcharts/)) for Python version >= 3.10.

Regarding the status of the issue, there is still work in progress:

* We have been looking for alternatives, but unfortunately at this moment there is no goo alternative.
* We are also trying to get in touch with the maintainers of the project. We would like to know if there are plans to migrate the project to new maintainers. We have no success yet either. Hopefully they respond soon.
* If none of the above works out we will work on a different solution.

For now, I think we need to make this more visible in the README.
I will make some changes on that later on.

Please let us know if you can work around for now using an older version of Python (or for example by installing one of the [python-highcharts forks](https://github.com/kyper-data/python-highcharts/network/members) which has fixes for Python 3.10 support)

With best regards,
Tom.

**Comment by sngx13** [comment]
Hi Tom,

Is there a way of running test using "byteblowerll.byteblower" instead? I can see "Highcharts" has PR open to get collections.abc implemented but the fix has been blocked: https://github.com/kyper-data/python-highcharts/pull/88

Thank you

**Comment by tomghyselinck** [comment]
This issue has been solved in version 1.0.0b14.

It has been released on March 29, 2023.

The original Python project is no longer maintained and the maintainer(s) are not reachable in any way.
We created a fork which addressed this issue and did some other code improvements: https://github.com/excentis/python-highcharts
